javascriptv3/example_code/secrets-manager/actions/get-secret-value.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {fileURLToPath} from "url";
+
+// snippet-start:[javascript.v3.secretsmanager.actions.GetSecretValue]
+import {
+  GetSecretValueCommand,
+  SecretsManagerClient,
+} from "@aws-sdk/client-secrets-manager";
+
+export const getSecretValue = async (secretName = "SECRET_NAME") => {
+  const client = new SecretsManagerClient();
+  const response = await client.send(
+    new GetSecretValueCommand({
+      SecretId: secretName,
+    })
+  );
+  console.log(response);
+  // {
+  //   '$metadata': {
+  //     httpStatusCode: 200,
+  //     requestId: '584eb612-f8b0-48c9-855e-6d246461b604',
+  //     extendedRequestId: undefined,
+  //     cfId: undefined,
+  //     attempts: 1,
+  //     totalRetryDelay: 0
+  //   },
+  //   ARN: 'arn:aws:secretsmanager:us-east-1:xxxxxxxxxxxx:secret:binary-secret-3873048-xxxxxx',
+  //   CreatedDate: 2023-08-08T19:29:51.294Z,
+  //   Name: 'binary-secret-3873048',
+  //   SecretBinary: Uint8Array(11) [
+  //      98, 105, 110, 97, 114,
+  //     121,  32, 100, 97, 116,
+  //      97
+  //   ],
+  //   VersionId: '712083f4-0d26-415e-8044-16735142cd6a',
+  //   VersionStages: [ 'AWSCURRENT' ]
+  // }
+
+  if (response.SecretString) {
+    return response.SecretString;
+  }
+
+  if (response.SecretBinary) {
+    return response.SecretBinary;
+  }
+};
```

Comment:
no default value/error response? What is the expected behavior if `SecretString` and `SecretBinary` both evaluate to false? Or is there some way we know that will never happen?

Reply:
If it's not a string and it's not binary then it's not found and throws an error. Pretty clear with "ResourceNotFoundException"